### PR TITLE
New design and new features for each button

### PR DESCRIPTION
**Pull request by davigrca**

### CODE REVIEW

**Comment by jcneally** [comment]
@davigrca The spacing is all over the place in the files, can we fix that up. 2 space indentation per level eg.
```
function() {
  // code
}

<div>
  <div>
  </div>
</div>

```

Also, can you move `crossfade.xd` into a folder like `designs`?

**Comment by davigrca** [comment]
alright

**Comment by davigrca** [comment]
pull request changes done

File: how-to-use.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<html>
```
Comment (jcneally) [inline how-to-use.html:1]: spacing is still messed up here and info page

File: index.html [hunk 2]
```diff
@@ -1,432 +1,503 @@
...
+
+      .selected{
+        background-color: #2C72C7;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="container">
+
+      <div class="main-page" style="display: flex;flex-direction: row">
```
Comment (jcneally) [inline index.html:116]: move these styles to the .main-page styles above

File: index.html [hunk 3]
```diff
@@ -1,432 +1,503 @@
...
+          <div class="details">
+            <div class="total">Total of Songs: </div>
+            <div class="duration">Playlist Duration: </div>
+          </div>
+      </div>
+
+        <div class="right-side">
+
+          <div class="top-nav">
+            <a href=how-to-use.html target="_blank" style="margin:28px">HOW TO USE</a>
```
Comment (jcneally) [inline index.html:143]: no inline styles. Add a class

File: index.html [hunk 4]
```diff
@@ -1,432 +1,503 @@
...
+            <a href=info.html target="_blank" style="margin:59px">INFO</a>
```
Comment (jcneally) [inline index.html:144]: no inline styles. Add a class

File: index.html [hunk 5]
```diff
@@ -1,432 +1,503 @@
...
+          </div>
+
+          <div class="right-buttons">
+
+            <button class="main-button create-button" onClick="createPlaylist();">Create Playlist</button>
+            <button class="main-button remove-song" style="background-color: #c63d3d" onClick="removeButton()">remove song</button>
```
Comment (jcneally) [inline index.html:150]: Move these to class styles, same as below

File: index.html [hunk 6]
```diff
@@ -1,432 +1,503 @@
...
+
+      var songList = [];
+      var selectedSongIndex;
+
+      //creating a function songDetail to check if it exists youtubeLink variable inside of div songDetail, if it does, so it returns id youtubeLink variable, it it doesnt, it reurns string empty "
+      var songDetail = (youtubeLink) => (`<div class="song-detail">${youtubeLink ? youtubeLink.id : ""} </div>`);
+
+      //add a song to push it onto that array and save the playlist
+      function addSong(song) {
+      var el = $(songDetail(song)).appendTo('.song-list');
```
Comment (jcneally) [inline index.html:426]: spacing. Function inside should be 2 spaces over

File: index.html [hunk 7]
```diff
@@ -1,432 +1,503 @@
...
+
+        //to select all song and then to remove the class from them
+                $('.song-detail').removeClass('selected');
+        $(this).addClass('selected'); //to select clicked element
+      });
+
+      }
+
+      function removeButton(){
+      //if you dont select any song, so the index isnt a integer, so you do anything, you dont remove anything
```
Comment (jcneally) [inline index.html:443]: spacing

File: index.html [hunk 8]
```diff
@@ -1,432 +1,503 @@
...
+      if(typeof(selectedSongIndex) !== "number") { return;
```
Comment (jcneally) [inline index.html:444]: Put return on a newline below

File: index.html [hunk 9]
```diff
@@ -1,432 +1,503 @@
...
+      //remove the song from the list
+      songList.splice(selectedSongIndex, 1);
+      savePlaylist();
+      selectedSongIndex = null;
+
+      }
+
+      function addSongButton(){
+
+      var el = $('.song-input');
```
Comment (jcneally) [inline index.html:458]: spacing

File: index.html [hunk 10]
```diff
@@ -1,432 +1,503 @@
...
+      //after you add a song, reset all the inputs back to the empty string
+      $(".song-id").val("");
+      $(".start-time").val("");
+      $(".end-time").val("");
+
+      }
+
+      function savePlaylist(){
+
+      window.localStorage.setItem('songList', JSON.stringify(songList));
```
Comment (jcneally) [inline index.html:483]: spacing

File: index.html [hunk 11]
```diff
@@ -1,432 +1,503 @@
...
+
+      }
+
+      function createSamplePlaylist() {
+      // Remove the first empty one
```
Comment (jcneally) [inline index.html:488]: spacing

File: index.html (hunk 2 above)
Comment (davigrca) [reply index.html:116]: ok

**Pull request merged**